.github/workflows/daily_modin_precommit.yml

```diff
@@ -215,6 +215,74 @@ jobs:
             .tox/.coverage
             .tox/coverage.xml
 
+  test-enable-cte-optimization:
```

Comment:
What's the long term plan? e.g., deprecate this job once CTE is enabled by default?

Reply:
yes